src/librustc_mir/transform/nll/subtype.rs

```diff
@@ -0,0 +1,89 @@
+use super::RegionIndex;
```

Comment:
Tidy error 😏 

```
[00:03:02] tidy error: /checkout/src/librustc_mir/transform/nll/subtype.rs: incorrect license
[00:03:02] tidy error: /checkout/src/test/mir-opt/nll/region-liveness-drop-no-may-dangle.rs:48: line longer than 100 chars
[00:03:03] some tidy checks failed
```

Reply:
Thanks @kennytm I've already fixed this in my local machine, will push in a while :)